pkg/sypgp/sypgp.go

```diff
@@ -593,8 +595,13 @@ func SearchPubkey(search, keyserverURI, authToken string) error {
 		Size: 256,
 	}
 
+	//set the machine readable output on
+	var options []string
+	options = append(options, "mr")
 	// Retrieve first page of search results from Key Service.
-	keyText, err := c.PKSLookup(context.TODO(), &pd, search, client.OperationIndex, true, false, nil)
+	keyText, err := c.PKSLookup(context.TODO(), &pd, search, client.OperationIndex, true, false, options)
+
+	keyText = reformatMachineReadableOutput(keyText)
```

Comment:
There's no point using `keyText` until you've checked `err != nil`. Suggest removing this after that check.